042-SAPOnAzure/Student/07-PowerQuery.md

```diff
@@ -0,0 +1,85 @@
+# Challenge 7: Analytical Dashboard Using SAP S/4 Hana DB
+[< Previous Challenge](./06-Start-Stop-Automation.md) - **[Home](../README.md)** 
+
+# Introduction  
+
+Contoso inc. is a manufacturing company planning to modernize its IT infrastructure. As a part of that efforts, they want to Generate self-service report from SAP HANA DB tables using Microsoft power BI desktop client and develop a self-service report for visualization and analysis. 
+
+ 
+
+# Description 
+    
+  
+  This exercise includes connecting power BI desktop client to HANA DB (using direct query with DB user and password), Extract data from the required tables (using semantic SQL query) and Generate self-service Job status report.
+  
+  A high level architectural diagram can be found below.
+  
+  
+![image](https://user-images.githubusercontent.com/81314847/115074617-eefa6580-9ec7-11eb-9eaa-ca09b29708cb.png)
+ 
+
+
+## Tips
+ 
+
+   * Download and install SAP HANA client for windows ( x64 -- Coach will provide software link to download)
+    
+   * Download Power  BI  desktop from windows power app platfrom 
```

Comment:
Please hypertext the "Download PowerBI" text to this link: https://www.microsoft.com/en-us/download/details.aspx?id=58494

Reply:
updated.